bgpd/bgp_linkstate_tlv.c

```diff
@@ -316,6 +364,16 @@ static int bgp_linkstate_nlri_node_decode(struct bgp_linkstate_type_node **pp,
 	return ret;
 }
 
+static void
+bgp_linkstate_nlri_node_encode(struct bgp_linkstate_type_node *p_node,
+			       struct stream *s)
+{
+	stream_putc(s, p_node->proto);
+	stream_putq(s, p_node->identifier);
+
+	bgp_linkstate_nlri_node_descriptor_encode(&p_node->local_node_descr, s);
+}
```

Comment:
unknown node TLVs should somehow be encoded.

Reply:
see https://github.com/FRRouting/frr/pull/12649#discussion_r1169925019